src/sh/python-lint/poetry-check.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+POETRY_SUBDIRECTORY=./src/python/hooks
+
+poetry --version >/dev/null 2>&1 || {
+	echo "Poetry is not installed. Please install poetry to continue."
+	exit 1
+}
+
+cd $POETRY_SUBDIRECTORY || exit 1
+
+poetry check --quiet || {
+	echo "Poetry check failed. Please run $(poetry install -C src/python/hooks) to continue."
+	exit 1
+}
+
+pre-commit --version >/dev/null 2>&1 || {
+	echo "Pre-commit is not installed. Please install pre-commit to continue."
+	exit 1
+}
```

Comment:
This can probably come out, since the parent pre-commit command will fail anyways if checks get to this point